azure-pipelines/ui-automation/msal-local-nightly.yml

```diff
@@ -0,0 +1,159 @@
+name: $(Build.BuildId)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger: none
+pr: none
+
+schedules:
+  - cron: "0 5 * * 1-6" # 5:00 AM UTC everyday Mon-Sat
+    displayName: Auth Client Android SDK dev build
+    branches:
+      include:
+        - dev
+    always: true
+
+variables:
+  engineeringProjectId: 'fac9d424-53d2-45c0-91b5-ef6ba7a6bf26'
+  azureSamplePipelineId: 1458
+  msalAutomationAppApk: msalautomationapp-local-BrokerHost-debug.apk
+  msalAutomationAppTestApk: msalautomationapp-local-BrokerHost-debug-androidTest.apk
+  azureSampleApk: AzureSample-local-debug.apk
+  firebaseTimeout: 45m
+  resultsHistoryName: Dev MSAL without Broker
+
+parameters:
+  - name: firebaseDeviceId
+    displayName: Firebase Device Id
+    type: string
+    default: blueline
+  - name: firebaseDeviceAndroidVersion
+    displayName: Firebase Device Android Version
+    type: number
+    default: 28
+
+stages:
+  # msalautomationapp
+  - stage: 'msalautomationapp'
+    displayName: Build MSAL Automation APKs
+    jobs:
+      - job: 'msalautomationapp'
+        displayName: Build and Publish MSAL Automation app
+        pool:
+          vmImage: ubuntu-latest
+        steps:
+          - checkout: self
+            clean: true
+            submodules: recursive
+            persistCredentials: True
+          - bash: |
+              echo "##vso[task.setvariable variable=ENV_VSTS_MVN_ANDROID_MSAL_ACCESSTOKEN]$(mvnAccessToken)"
+            displayName: Set MVN Access Token in Environment
+          - task: AzureKeyVault@2
+            displayName: 'Get Key vault AndroidAutomationRunnerAppSecret'
+            inputs:
+              azureSubscription: 'MSIDLABS_ANDROID_KV'
+              KeyVaultName: 'ADALTestInfo'
+              SecretsFilter: 'AndroidAutomationRunnerAppSecret'
+              RunAsPreJob: false
+          - task: Gradle@2
+            displayName: 'Assemble MSAL Automation App'
+            inputs:
+              tasks: clean msalautomationapp:assembleLocalBrokerHostDebug -PlabSecret=$(AndroidAutomationRunnerAppSecret) -PbrokerSource=LocalApk
+              publishJUnitResults: false
+          - task: Gradle@2
+            displayName: 'Assemble MSAL Automation App Instrumented Tests'
+            inputs:
+              tasks: msalautomationapp:assembleLocalBrokerHostDebugAndroidTest -PlabSecret=$(AndroidAutomationRunnerAppSecret)
+              publishJUnitResults: false
+          - task: CopyFiles@2
+            displayName: 'Copy apks for later use in the pipeline'
+            inputs:
+              flattenFolders: true
+              contents: '$(Build.SourcesDirectory)/msalautomationapp/build/outputs/apk/**/*.apk'
+              targetFolder: '$(Build.ArtifactStagingDirectory)/msal'
+          - publish: '$(Build.ArtifactStagingDirectory)/msal'
+            displayName: 'Publish apks for later use'
+            artifact: msalautomationapks
+  # Brokers
+  - stage: 'brokers'
```

Comment:
Update the stage name to AzureSample

Reply:
I think you need to update the stage and job names in multiple place in this file.. to remove broker references

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1644/commits/3243d502eb6ea8d4072a5ad52d988839e689d5e1 and here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1644/commits/eab8a801bf9b479971ba46a1b3a63786cfb6cf9f